test-clients/src/main/java/com/hedera/services/yahcli/commands/accounts/SendCommand.java

```diff
@@ -73,9 +73,17 @@ public class SendCommand implements Callable<Integer> {
     public Integer call() throws Exception {
         var config = configFrom(accountsCommand.getYahcli());
 
+        if (!config.allowListExist(Long.parseLong(beneficiary))) {
+            throw new CommandLine.ParameterException(
+                    accountsCommand.getYahcli().getSpec().commandLine(),
+                    "Beneficiary " + beneficiary + " supposed to be in allow list");
+        }
+
         long amount;
+        String originaldomination = denomination;
```

Comment:
Spelling error. It should be original**Denom**ination

Reply:
thank will fix it

Reply:
fixed

Reply:
I kinda like 'Original Domination' 🤔